javatests/arcs/android/entity/BUILD

```diff
@@ -8,6 +8,7 @@ licenses(["notice"])
 arcs_kt_android_test_suite(
     name = "entity",
     size = "medium",
+    flaky = True,
```

Comment:
If we do this, we should consider tagging all `= True` with a tracking bug?

Reply:
+1

Reply:
Done